tools/submission/submission_checker.py

```diff
@@ -735,19 +735,19 @@
 ACC_PATTERN = {
     "acc": r"^(?:\{\"accuracy|accuracy)[\": ]*=?\s*([\d\.]+).*",
     "AUC": r"^AUC=([\d\.]+).*",
-    "mAP": r".*'(?:mAP|Total)':\s*([\d\.]+)",
+    "mAP": r".*(?:mAP|Total)=([\d.]+)%",
```

Comment:
https://github.com/mlcommons/mlperf_inference_unofficial_submissions_v5.0/blob/main/open/MLCommons/results/mlc-server-reference-gpu-pytorch_v2.2.2-cu124/pointpainting/singlestream/accuracy/accuracy.txt

We need ":" for automotive right?

Reply:
shall we do `"mAP": r".*(?:mAP=|Total:)\s*([\d.]+)"` ?

Reply:
Let me change it